src/linker/Linker.Steps/LinkAttributesParser.cs

```diff
@@ -281,8 +280,6 @@ CustomAttributeArgument[] ReadCustomAttributeArguments (XPathNodeIterator iterat
 
 				return new CustomAttributeArgument (typeref, type);
 			default:
-				// TODO: Add support for null values
```

Comment:
I'd keep it, these TODOs are still valid

Reply:
They are - but they're basically a feature request. I'm just not a fan of TODOs in merged code - it's either "bug", "future feature" or "nice to have". But on their own they're just confusing - when I find one, I always try to figure out which one it is and if I should care or not.